lib/bracket-matcher.coffee

```diff
@@ -40,14 +41,14 @@ class BracketMatcher
     @subscriptions.add atom.commands.add editorElement, 'bracket-matcher:remove-brackets-from-selection', (event) =>
       event.abortKeyBinding() unless @removeBrackets()
 
-    @subscriptions.add atom.config.observe 'bracket-matcher.autocompleteSmartQuotes', (newValue) =>
```

Comment:
If we go ahead with the change proposed in https://github.com/atom/bracket-matcher/pull/223/files#r57720335, we might be able to keep this, and toggle quotes only when either the related configuration or the grammar change. What do you think?